### PR TITLE
Add Windows ColorizedStderrHandler support

### PR DESCRIPTION
**Pull request by RazerM**
This address #98.

I also fixed some documentation errors.

### CODE REVIEW

File: logbook/more.py [hunk 1]
```diff
@@ -323,7 +327,22 @@ class ColorizedStderrHandler(ColorizingStreamHandlerMixin, StderrHandler):
     not colorize on Windows systems.
 
     .. versionadded:: 0.3
+    .. versionchanged:: TODO
```
Comment (RazerM) [inline logbook/more.py]: Note that this needs replace with a version number before release!

Comment (vmalloc) [inline logbook/more.py]: I think you can safely write 1.0.0 there.

File: logbook/utils.py [hunk 2]
```diff
@@ -56,6 +58,8 @@ def suppressed_deprecations():
 
     >>> with suppressed_deprecations():
     ...    call_some_deprecated_logic()
+
+    .. versionadded:: 12.0
```
Comment (vmalloc) [inline logbook/utils.py]: that's 0.12

File: logbook/utils.py [hunk 3]
```diff
@@ -157,7 +161,9 @@ def deprecated(func=None, message=None):
     ...     pass
 
     This will cause a warning log to be emitted when the function gets called,
-    with the correct filename/lineno
+    with the correct filename/lineno.
+
+    .. versionadded:: 12.0
```
Comment (vmalloc) [inline logbook/utils.py]: 0.12

**Comment by vmalloc** [comment]
Thanks!

**Pull request merged**